doc/edl/README.md

```diff
@@ -3,16 +3,30 @@
 ## Background
 
 A PaddlePaddle training job contains several trainer instances,
-several parameter server instances, and one master instance. We would
-like to automatically scale the number of training instances and the
+several parameter server instances, and one master instance. 
+We are going to define PaddlePaddle cluster job using Kubernetes [CRD](https://kubernetes.io/docs/concepts/api-extension/custom-resources/). 
+Each job is described using a yaml representation called TrainingJob. 
+With CRD, we can manage the PaddlePaddle cluster job as easy as k8s primary resources. 
+We also would like to automatically scale the number of training instances and the
 number of parameter server instances to fully utilize the cluster's
 computation resources. We call this Elastic Deep Learning.
 
+Currently, we have two controller Lifecycle Controller and Autoscale Controller to support the functions mentioned 
+above. We will merge them in the near future.
+
+**The goals of Lifecycle Controller including:**
+
+- Implement a controller and register a TrainingJob CRD
+- Create and delete TrainingJob
+- TrainingJob life cycle management
+
+**The goals of Autoscale Controller including:**
```

Comment:
We can call the controller "controller", and the `Autoscaler` is not a Kubernetes controller actually, It's just a goroutine running at the background to do autoscaling.